ibm/service/atracker/data_source_ibm_atracker_targets.go

```diff
@@ -125,27 +125,34 @@ func DataSourceIBMAtrackerTargets() *schema.Resource {
 								},
 							},
 						},
-						"cos_write_status": {
```

Comment:
Why is this attribute removed "cos_write_status"
Please follow Hashicorp Terraform standarad process if you want to deprecate or remove an attribute/argument
https://developer.hashicorp.com/terraform/plugin/sdkv2/best-practices/deprecations

Reply:
Added it back